### PR TITLE
DOC-5959 platform help for PHP 4.0 (comments welcome!)

### PR DESCRIPTION
**Pull request by osfameron**
Some of the specific details for 4.0 may be in flux, but please do comment on anything, for example:

* installation details (can they be made smoother for Linux / Windows etc.?)
* should we document PhpStorm also?
* NB: though I've got 4.0 installed now (thanks @avsej !) I can't get php to see the 4.0 library yet, which means I'm not sure if the VS Code intelephense plugin works nicely yet.

### CODE REVIEW

File: modules/hello-world/pages/platform-help.adoc [hunk 1]
```diff
@@ -0,0 +1,172 @@
+= Setting Up Couchbase PHP SDK with VSCode
+:description: pass:q[Discover how to get up and running developing applications with the Couchbase PHP SDK 3.0+ using `Visual Studio Code`.]
```
Comment (RichardSmedley) [inline modules/hello-world/pages/platform-help.adoc]: Are we adding this to the 3.x docs as well, then?

File: modules/hello-world/pages/platform-help.adoc [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+:page-aliases: ROOT:platform-introduction,ROOT:platform-help,php-environment
```
Comment (RichardSmedley) [inline modules/hello-world/pages/platform-help.adoc]: Not sure if these have all been used, a quick look at https://docs-archive.couchbase.com/php-sdk/2.1/overview.html isn't turning up anything

File: modules/hello-world/pages/platform-help.adoc [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
++
+--
+https://www.php.net/manual/en/install.unix.php has information on installing on various Linux systems.
+Where there are no specific instructions on the official PHP page, or in our tested guidance below, it is worth consulting your distro's trusted documentation, as there may be known-good repositories with PHP binaries to evaluate.
+
+Once you have installed PHP, simply use `pecl` to install the Couchbase PHP SDK.
+
+[source,console]
+----
+pecl install https://packages.couchbase.com/clients/php/couchbase-4.0.0.tgz
```
Comment (RichardSmedley) [inline modules/hello-world/pages/platform-help.adoc]: I thought we were sticking with $ prompt for BASH?

File: modules/hello-world/pages/platform-help.adoc (hunk 1 above)
Comment (osfameron) [reply modules/hello-world/pages/platform-help.adoc]: oops

File: modules/hello-world/pages/platform-help.adoc (hunk 2 above)
Comment (osfameron) [reply modules/hello-world/pages/platform-help.adoc]: copy/pasta from the .NET one... will remove

File: modules/hello-world/pages/platform-help.adoc (hunk 3 above)
Comment (osfameron) [reply modules/hello-world/pages/platform-help.adoc]: Oh, er, possibly. The copy/paste of `$` doesn't feel great, let's discuss, happy to push either way to be consistent 👍

**Review by RichardSmedley — APPROVED** [review]

**Comment by osfameron** [comment]
Closing in favour of #95

**Pull request closed without merging**